src/mudlet.h

```diff
@@ -258,6 +258,7 @@ class mudlet : public QMainWindow, public Ui::main_window
     void stopSounds();
     void playSound(const QString &s, int);
     QStringList getAvailableFonts();
+    const QStringList& getDefaultGames() const { return mDefaultGames; }
```

Comment:
If you make the list `static const` then this can be a `static` method...

Reply:
`mudlet.h:656:30: error: static data member of type 'const QStringList' must be initialized out of line`... doesn't seem like worth the trouble

Reply:
There is a magical way of doing it in the header file - lemme see if I can find it...

Reply:
Ah, yeah, the magic word is `inline`:
```cpp
    inline static const QStringList scmDefaultGames {QStringLiteral("3Kingdoms"),
                                                     QStringLiteral("3Scapes"),
                                                     QStringLiteral("Aardwolf"),
                                                     QStringLiteral("Achaea"),
                                                     QStringLiteral("Aetolia"),
                                                     QStringLiteral("Avalon.de"),
                                                     QStringLiteral("BatMUD"),
                                                     QStringLiteral("Clessidra"),
                                                     QStringLiteral("Fierymud"),
                                                     QStringLiteral("Imperian"),
                                                     QStringLiteral("Luminari"),
                                                     QStringLiteral("Lusternia"),
                                                     QStringLiteral("Materia Magica"),
                                                     QStringLiteral("Midnight Sun 2"),
                                                     QStringLiteral("Realms of Despair"),
                                                     QStringLiteral("Reinos de Leyenda"),
                                                     QStringLiteral("StickMUD"),
                                                     QStringLiteral("WoTMUD"),
                                                     QStringLiteral("ZombieMUD"),
                                                     QStringLiteral("Carrion Fields"),
                                                     QStringLiteral("Cleft of Dimensions"),
                                                     QStringLiteral("CoreMUD"),
                                                     QStringLiteral("God Wars II"),
                                                     QStringLiteral("Slothmud"),
                                                     QStringLiteral("Legends of the Jedi")};

```
Since that is an immutable constant `QStringList` that can be made `public:` and used directly - without the need to have a method to return a `const &`...

Reply:
Refer to https://github.com/vadi2/Mudlet/pull/55 on your repository to see the full changeset...